configuration-sample/ods-core.env.sample

```diff
@@ -338,6 +341,9 @@ JIRA_PROJECT_TEMPLATE_KEY_NAMES=${project.template.default.key},kanban
 # e.g: DEFAULT_OPENSHIFT_PERMISSION_GROUPS="ADMINGROUP=MY-DEFAULT-CLUSTER-ADMIN-GROUP"
 DEFAULT_OPENSHIFT_PERMISSION_GROUPS=
 
+# Enable or disable the openshift service in the provisioning app (values: "true" or "false")
```

Comment:
wrong comment, or? . it's about enabling verification of if a project exits

Reply:
No, the comment is correct... this property disables or enables the service in prov app. Similar switch was implemented to enable or disable the confluence adapter (which is disabled in odsbox because there is no confluence running there)

Reply:
ahh - this is a much larger change now, not just the pre-check .... hmmmm

Reply:
mmm, like the JiraAdapter and other adapters in ProvApp, the OpenshiftService encapsulates the logic related to the functionality that it provides, which is the pre-check.